README.md

```diff
@@ -15,14 +15,14 @@ look something like:
 
 ```
 cd ~/"Library/Application Support/Sublime Text 3/Packages"
-git clone https://github.com/Microsoft/TypeScript-Sublime-Plugin.git TypeScript
+git clone --depth=1 https://github.com/Microsoft/TypeScript-Sublime-Plugin.git TypeScript
```

Comment:
Add a note to explain what ``--depth=1`` does. Something like this:
(``--depth=1`` downloads only the current version to reduce the clone size)